### DISCUSSION TITLE
From Raw Material (Production) To Be A Product

### DISCUSSION THREAD

**Discussion by fahmiegerton** [Blair2004/NexoPOS#1968] [Q&A] [answered]
Sorry for my bad English, I have a case for a fruit/vegetable juice store, they need to be able to convert an ingredient of fruit into a product, for example:
- A product called "120ml of Avocado Juice" was made by 30gr of avocado (cost 5 cents), 0.08L of milk (cost 7 cents), and 0.1L of water (cost 2 cents).

How do I combine that ingredient as raw material into a product? I need this because I need to "transfer" the COGS of the ingredients into the COGS of a product so I can decide the cost of sale of the product, if it possible with NexoPOS.

Thank you in advance for the help!

**Comment by Blair2004** [comment, marked as answer]
Hi, this is supposed to be implemented on the Raw Material module.
It's on progress.

**Reply by fahmiegerton** [reply]
Thanks! As for now, I just use grouped products instead. but I saw it's already released, is there any demo I could take a look at?

**Reply by Cotopaco1** [reply]
I also considered using grouped products, but I had trouble determining the available stock for the grouped product itself.
For example, if I have a "Burger" as a grouped product, and it includes two material items like bread and meat, I can see the stock of the individual ingredients, but the system doesn’t tell me how many burgers I can make based on the current stock.

@fahmiegerton Did you run into this issue as well?
